.env.example

```diff
@@ -22,10 +22,12 @@ MAIL_USERNAME=null
 MAIL_PASSWORD=null
 MAIL_ENCRYPTION=null
 
-NORTHSTAR_URL=https://northstar-qa.dosomething.org
+NORTHSTAR_URL=http://northstar.dev:8000
 NORTHSTAR_API_KEY=abc4324
 
 FASTLY_KEY=abc123
 FASTLY_TABLE_REDIRECTS=abc123
 FASTLY_TABLE_REDIRECT_TYPES=abc123
-FASTLY_SERVICE_KEY=abc123
\ No newline at end of file
+FASTLY_SERVICE_KEY=abc123
+
+SERVICE_DOMAIN=example.com
```

Comment:
@angaither yeah it's the domain that the fastly service works on (so eventually dosomething.org). Is `FASTLY_SERVICE_DOMAIN` clearer?